### PR TITLE
Update info about Django, Flask and Python

### PR DESCRIPTION
**Pull request by Krejdom**
Hi, I've added new tutorials for Django, Flask and Python and updated info in these sections.

### CODE REVIEW

**Comment by strzibny** [comment]
Even though it's tempting I would like to avoid changing permanent URL's that are already in use and indexed/linked. In future it might be okay to look back and rename something (with appropriate fallback).

File: tech/languages/python/django-installation.md [hunk 1]
```diff
@@ -4,20 +4,65 @@ subsection: python
...
 
-For Python 3:
+And run the server.
+```bash
+$ python manage.py runserver
+```
+
+Now that the server’s running, visit http://127.0.0.1:8000/ with your Web browser. You willl see a “Welcome to Django” page, in pleasant, light-blue pastel. It worked!
+
+## DevAssistant for Django
```
Comment (hroncok) [inline tech/languages/python/django-installation.md]: Please don't promote DevAssistant any more, it's dead.

**Comment by strzibny** [comment]
I would suggest one last change -- keep all headlines as nouns (Running vs Run, installation vs install).

File: tech/languages/python/python-installation.md [hunk 2]
```diff
@@ -1,37 +1,55 @@
...
 
+```bash
+$ sudo dnf install python3-bpython
 ```
-$ sudo dnf install pypy
+
+Now you can run _bpython_. For Python 3 version type `python3-bpython`. You should see something similar:
+
+```python
+bpython version 0.15 on top of Python 3.5.1 /usr/bin/python3
```
Comment (strzibny) [inline tech/languages/python/python-installation.md]: This is a type I assume.

**Comment by strzibny** [comment]
This looks okay from my point of view, thanks @Krejdom

**Pull request merged**